meetingroommanagement/src/main/java/com/hsbc/meets/dao/impl/HomeJdbcDaoImpl.java

```diff
@@ -27,19 +31,16 @@
  *
  */
 public class HomeJdbcDaoImpl implements HomeDao {
-
-	private static final String SELECT_USERS_BY_NAME_SQL = "SELECT ID, Name, Email FROM users WHERE Name LIKE ?";
-	private static final String CHECK_ROWS_IN_USERS_SQL = "SELECT COUNT(*) FROM users";
-	private static final String INSERT_USERS_DATA_SQL = "INSERT INTO users (ID, Name, Email, Phone, Credits, Role, Password) VALUES(?,?,?,?,?,?,?)";
 	private static final String XML_FILE_PATH = "src/main/webapp/resources/users.xml";
+	static Logger logger = LoggerFactory.getLogger();
 

```

Comment:
You changed the `PreparedStatement` to `CallableStatement` but you aren't using the stored procedures

Reply:
Done